clangd/ClangdServer.h

```diff
@@ -217,7 +219,7 @@ class ClangdServer {
                llvm::Optional<StringRef> ResourceDir = llvm::None);
 
   /// Set the root path of the workspace.
-  void setRootPath(PathRef RootPath);
+  void setRootPath(PathRef RootPath, std::vector<Path> ExclusionList);
```

Comment:
Maybe let's rename this to setConfiguration? and pass a ClangdConfigurationParamsChange